.github/workflows/deploy.yml

```diff
@@ -0,0 +1,53 @@
+name: Deploy
+
+on:
+  push:
+    branches:
+      - develop
+      - master
+
+jobs:
```

Comment:
These should both be gated on something like `if: github.repository_owner == 'dequelabs'` to avoid the possibility of forks accidentally trying to run them